sklearn/metrics/classification.py

```diff
@@ -221,8 +221,10 @@ def confusion_matrix(y_true, y_pred, labels=None, sample_weight=None):
 
     References
     ----------
-    .. [1] `Wikipedia entry for the Confusion matrix
+    .. [1] `Wikipedia entry for the Confusion matrix*
            <https://en.wikipedia.org/wiki/Confusion_matrix>`_
+           *NOTE: Different conventions exist for confusion matrix axes.
```

Comment:
I wouldn't use a * like this. It took me a while to understand. We could state it like this. Do you not think a Notes section would be more appropriate? Otherwise, I suppose something here saying "Wikipedia may use a different convention" doesn't hurt.

Reply:
How about making the wikipedia note parenthetical?
```python 
"""
    References
    ----------
    .. [1] `Wikipedia entry for the Confusion matrix
           <https://en.wikipedia.org/wiki/Confusion_matrix>`_
           (Wikipedia may use a different convention for axes)
"""